.ci/setup_kong.sh

```diff
@@ -8,7 +8,7 @@ sudo apt-get update
 sudo apt-get install wget curl tar make gcc unzip git liblua5.1-0-dev
 
 # Installing dependencies required for Kong
-sudo apt-get install sudo netcat lua5.1 openssl libpcre3 dnsmasq
+sudo apt-get install sudo netcat lua5.1 openssl libpcre3 dnsmasq libuuid1 uuid-dev
```

Comment:
Just to be sure, and because builds are already really slow: is `libuuid1` really needed here? In Kong-Vagrant we only use `uuid-dev`. I believe `libuuid1` depends on `uuid-dev` but not the other way around.